src/Common/src/System/Diagnostics/Debug.Windows.cs

```diff
@@ -75,6 +77,25 @@ private static void WriteToDebugger(string message)
                     Interop.mincore.OutputDebugString(message ?? string.Empty);
                 }
             }
+
+            private static unsafe bool WriteToStdErr(string message)
+            {
+                IntPtr handle = Interop.mincore.GetStdHandle(Interop.mincore.HandleTypes.STD_ERROR_HANDLE);
+                if (handle != IntPtr.Zero && handle != new IntPtr(-1))
+                {
+                    byte[] messageBytes = Encoding.ASCII.GetBytes(message);
```

Comment:
What's the behavior of this if I have non ASCII characters in the string?  Does the default fallback just put in '?'